src/main/java/com/fasterxml/jackson/databind/BeanDescription.java

```diff
@@ -173,6 +173,17 @@ public boolean isNonStaticInnerClass() {
     /**********************************************************
      */
 
+    /**
+     * Method for locating accessor (readable field, or "getter" method)
+     * that has
+     * {@link com.fasterxml.jackson.annotation.JsonKey} annotation,
+     * if any. If multiple ones are found,
+     * an error is reported by throwing {@link IllegalArgumentException}
+     *
+     * @since TODO
+     */
+    public abstract AnnotatedMember findJsonKeyAccessor();
```

Comment:
I know that some additions have been left as `abstract`, but for maximum backwards-compatibility, let's add "return null;" as implementation just in case.